Models/Circle_Model.py

```diff
@@ -16,7 +16,7 @@ def __BaseSetup(self,size,color:tuple,location:tuple,__defaultSurface):
     def Display(self):
         pygame.draw.circle(self.__DefaultSurface,self.Color,self.Location,self.Size)
 
-    def MoveTo(self, moveX:tuple,moveY:tuple):
+    def MoveTo(self, moveX,moveY):
```

Comment:
to keep consitant with the changes ive made name the movex and movey to x_loc and y_loc respectively